model/docs/pyrenew_demo.qmd

```diff
@@ -199,4 +218,6 @@ for samp_id in samp_ids:
 ax.set_ylim([0.4, 1/.4])
 ax.set_yticks([0.5, 1, 2])
 ax.set_yscale("log")
+
+# Can we use ArviZ for visualization? It is included in poetry dependencies.
```

Comment:
What's that? If you think it improves things, for sure!

Reply:
https://python.arviz.org/en/stable/index.html

It is included [here](https://github.com/CDCgov/multisignal-epi-inference/blob/437016d9e092587b3bf6e8469786bd54cf8e55a2/model/poetry.lock#L1310), so it seems that someone else may have been keen to use it.